### PR TITLE
Add Rego doc and edit Caching doc

### PR DESCRIPTION
**Pull request by ltagliaferri**
Adding rego policies doc

Signed-off-by: ltagliaferri

### CODE REVIEW

**Review by SharpRake — CHANGES_REQUESTED** [review]
Some minor tweaks, plus one paragraph where I had a slightly different interpretation of Adam's blog post. LMK if you have any questions on anything!

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+* `includeObjectMeta:` allows you to access the fields in the metadata: portion of the manifest, including the object’s name and labels.
+* `includeTypeMeta:` allows access to the top level fields in the manifest, such as the `kind` and `apiVersion`.
+* `fetchConfigFile:` fetches the OCI config file from the registry, which contains metadata about the image in the registry.
+
+Rego policies in Enforce must specify `type: rego` and the `data` field must contain `package sigstore`. 
+
+For the policy to pass, the `isCompliant` field must evaluate to `true` within the curly braces. The `isCompliant` Boolean is set to false by default, and the logic in the braces must flip the boolean to true for the policy to pass. 
+
+Within the `isCompliant` braces, multiple conditions may exist if they are added together with the `AND` keyword Multiple evaluations within different sets of braces may also exist in the policy, these will be chained together with the `OR` keyword, meaning that if any of them is true, the Boolean evaluate to be true.
```
[the remark this answers is not in the document]
Comment (ltagliaferri) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md:69]: May be worth calling out that Rego has this functionality that is not in the box with Enforce

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/annotation-based-caching.md [hunk 2]
```diff
@@ -13,26 +13,43 @@ weight: 100
...
+
+This feature will soon be the default for all users. In the mean time, in order to try out this feature on your Kubernetes cluster, you can use a feature flag setting in the ConfigMap `config-policy-controller` within the `cosign-system` namespace.
+
+## Understanding Annotation-based Caching
+
+Because a Kubernetes Pod usually receives most of its images passed down from parent objects like `StatefulSets` or `ReplicaSets`, enabling annotation-based caching allows for the verification results from parent objects to be passed down to Pods. This reduces the traffic made to container registries, reducing latency and rendering policy enforcement a lot more scalable.
+
+In Chainguard's own load testing annotation-based caching, with a cache hit we see P99 verification latency of under 2s, and no egress was made to container registry. Without caching, latency would be highly dependent of registry access latency, where the P99 latency may be 10s or more.
```
[the remark this answers is not in the document]
Comment (ltagliaferri) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/annotation-based-caching.md:26]: I added a parenthetical explanation of P99, see https://stackoverflow.com/questions/12808934/what-is-p99-latency

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-kubernetes-faq.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ weight: 800
 toc: true
 ---
 
+> _This document relates to Chainguard Enforce. In order to follow along, you will need access to Chainguard Enforce. You can request access through selecting **Chainguard Enforce for Kubernetes** on the [inquiry form](https://www.chainguard.dev/get-demo?utm_source=docs)._
```
[the remark this answers is not in the document]
Comment (ltagliaferri) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-kubernetes-faq.md:16]: I guess we should do it

**Review by SharpRake — APPROVED** [review]
Just a couple very minor suggestions. Otherwise this looks great!

**Pull request merged**